config/initializers/oracle.rb

```diff
@@ -23,6 +23,7 @@ def column(name, type, options = {})
   ActiveRecord::ConnectionAdapters::OracleEnhancedAdapter.class_eval do
     remove_const(:IDENTIFIER_MAX_LENGTH)
     const_set(:IDENTIFIER_MAX_LENGTH, 128)
+    remove_method(:log)
```

Comment:
If you prepend no need to remove the method no?
But that's fine

Reply:
We need to remove it because it calls super and we cannot pass through the old implementation.